app/views/layouts/_navbar.html.erb

```diff
@@ -1,9 +1,16 @@
 <%# https://getbootstrap.com/docs/4.5/components/navbar/ %>
 <nav class="navbar navbar-expand-md navbar-dark bg-dark">
   <div class="navbar-brand"><%= link_to I18n.t('index.portal_name'), root_path, class: 'portal-link' %></div>
-  <form class="form-inline">
-    <i class="fa fa-bell" style="color: white; padding: 10px"></i>
-    <a href="<%= search_users_path %>"><i class="fa fa-search" style="color: white; padding: 10px"></i> </a>
-  </form>
+  <% if user_signed_in? %>
+    <form class="form-inline">
+      <% if current_user.contact_requests.any? %>
+        <a href="<%= user_contact_requests_path(current_user) %>"> <i class="fa fa-bell" style="color: red; padding: 3px"></i> </a>
```

Comment:
I would extract all styles into an appropriate css file

Reply:
I wouldn't like to do that, because if someone else uses the bell icon, they don't necessarily want my styling...

Reply:
Yes but you could use a class like `notification-bell` or something like that. I just don't like to have HTML, CSS and application logic all in one file :sweat_smile: But if you prefer the inline styling that's fine for me because it's not such a big issue :+1:

Reply:
I agree with @felixauringer. If someone doesn't like the styling, the class can be changed even easier